pytorch_lightning/metrics/functional/classification.py

```diff
@@ -134,10 +134,38 @@ def stat_scores(
     return tp, fp, tn, fn, sup
 
 
+def _get_topk(
+        pred: torch.Tensor,
+        target: torch.Tensor,
+        topk: Optional[int] = None,
+) -> torch.Tensor:
+    if (target.ndim > 1) and (topk is not None):
+        raise ValueError(
+            f'topk={topk} is not supported for multi-label target values.'
+        )
+
+    if (target.ndim > 1) or (pred.ndim == 1):
```

Comment:
I don't fully understand this case... Can you explain, why to return the whole pred if `target.ndim>1`?

Reply:
multi-label classification with `topk=None`. `topk` doesn't work for multi-label classification right?